templates/default/disable_ipv6_loopback.sh.erb

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# Disable the ipv6 loopback address to make chef server and EMPD happy
+if ! grep net.ipv6.conf.lo.disable_ipv6 /etc/sysctl.conf; then
+  echo "net.ipv6.conf.lo.disable_ipv6=0" | sudo tee -a /etc/sysctl.conf
```

Comment:
Minor nit-pick, since this is `disable_ipv6` technically setting it to 0 should be /enabling/ ipv6 on lo.  Unless I'm mis-understanding, which is completely possible.